src/logger_dispatch_overrides.jl

```diff
@@ -81,6 +81,49 @@ struct TBImage <: WrapperLogType
     format::ImageFormat
 end
 content(x::TBImage) = x.data
+
+function Base.convert(T::Type{PNG}, tb_image::TBImage)
+    imgArray = tb_image.data
+    format   = tb_image.format
+
+    #unpack RGB, RGBA value to channels using channelview
+    imgArray = channelview(imgArray)
+    #if data contains integer numbers, scale them to 0-255
+    if isa(first(imgArray), Integer)
+        imgArray = (imgArray./255)
+    end
+    #convert all values to `Float64` for uniformity
+    imgArray = Float64.(imgArray)
+    #scale all values to 0-1
+    imgArray = (imgArray./(max(maximum(imgArray), 1)))
+    #convert any format to CHW
+    imgArray =
+        format == L   ? reshape(imgArray, (1, 1, size(imgArray, 1))) :
+        format == CL  ? reshape(imgArray, (size(imgArray, 1), 1, size(imgArray, 2))) :
+        format == LC  ? reshape(transpose(imgArray), (size(imgArray, 2), 1, size(imgArray, 1))) :
+        format == HW  ? reshape(imgArray, (1, size(imgArray, 1), size(imgArray, 2))) :
+        format == WH  ? reshape(transpose(imgArray), (1, size(imgArray, 2), size(imgArray, 1))) :
+        format == HWC ? permutedims(imgArray, (3, 1, 2)) :
+        format == WHC ? permutedims(imgArray, (3, 2, 1)) :
+        format == CHW ? imgArray :
+        format == CWH ? permutedims(imgArray, (1, 3, 2)) :
+    #== else ==#    throw("Invalid format")
+    channels, height, width = size(imgArray)
+    color =
+    channels == 1 ? Gray :
+    channels == 2 ? GrayA :
+    channels == 3 ? RGB :
+    channels == 4 ? RGBA :
```

Comment:
I think maybe you forgot to
Indent the 4 above lines?
`ImgArray` does it right